src/myr/reference.js

```diff
@@ -205,6 +205,14 @@ let transformations = [
   //  description: <span>Prevents the <span className="string">element</span> from being pushed by the cursor, but allows the element to be affected by physics.</span>
   //},
   //Not quite sure about the inner workings of this one yet
+  {
+    name: <span>makeSubtractive(<span className="string">element</span>)</span>,
+    description: <span>Makes the <span className="string">element</span> erase other shapes that it overlaps with, instead of being drawn as a new shape.</span>
```

Comment:
Is there a better word than erase?

Reply:
i was going to put "subtract from" but i wasn't sure how clear that would be.

Reply:
Let's make it subtract from. It's not really erasing. We made the reference examples for if someone is confused.

Reply:
will do!